resources/osx/preinstall.erb

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 
-# If we appear to be in an upgrade unload services.
+# Cleanup previous operations
+if [ -e "/tmp/.<%= @name %>.upgrade" ]
+  rm -rf "/tmp/.<%= @name %>.upgrade"
```

Comment:
Yep, made it a directory (see the preinstall), then touch a file for each service found by the preinstall